docs/redoc/master/openapi.json

```diff
@@ -12000,6 +12007,60 @@
           }
         }
       },
+      "ShardCleanStatusTelemetry": {
+        "oneOf": [
+          {
+            "type": "string",
+            "enum": [
+              "started",
+              "done",
+              "cancelled"
+            ]
+          },
+          {
+            "type": "object",
+            "required": [
+              "progress"
+            ],
+            "properties": {
+              "progress": {
+                "type": "object",
+                "required": [
+                  "deleted_points"
+                ],
+                "properties": {
+                  "deleted_points": {
+                    "type": "integer",
+                    "format": "uint",
+                    "minimum": 0
+                  }
+                }
+              }
+            },
+            "additionalProperties": false
+          },
```

Comment:
I have suspicion that the python client will struggle to generate these anonymous variants. Might be better to just add extra structs for these cases

Reply:
Hmm. These types are exclusively used in the telemetry API though, and not really intended to be parsed/used directly by our users. I therefore think it's fine to have them like this.

@generall Thoughts?

Reply:
I shortly mentioned this in our daily. I'll add proper types here. I have a separate time for internal vs telemetry anyway, so this is easy to pull off.

It's not strictly necessary as a user will never use this as input, but it's easy to fix so I'll implement it anyway.

Reply:
Fixed in <https://github.com/qdrant/qdrant/pull/5677/commits/60d86d1953161187a7ea62f7afdf8de431b69ccd>